docs/integrations/aws/index.md

```diff
@@ -32,8 +32,92 @@ In additional to runtime libraries installed by the Serverless Extension, an ext
 Lambda Extension is used to forward logs, metrics, and traces to
 Serverless Console. This component is independent from the language runtime.
 
-### Configuring the Serverless Console Extension
+## Configuring the Serverless Console Extension
 [Serverless Framework](../index.md) is the easiest way to instrument your app or
 service with the Serverless Console Extension. If your not using 
 Serverless Framework you'll need to build, and deploy the extension 
 yourself for each of your Lambdas functions. 
+
+### Pre-requisites
+Before you get started make sure you have the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html)
+and a recent version of [Node.js](https://nodejs.dev/learn/how-to-install-nodejs) and a 
+[Serverless Console Organization](https://console.serverless.com?ref_website=https%3A%2F%2Fwww.serverless.com%2Fconsole%2Fdocs%2F) and [jq](https://stedolan.github.io/jq/download/).
+
+### 1.Clone the repo and install Node modules
+Once you have everything you need you'll need to clone this repo
+and install node modules.
+
+```text
+git clone https://github.com/serverless/console.git
+cd console
+npm install
```

Comment:
This installs the main dev dependencies of the repository, but won't install the dependencies for the `aws-lambda-otel-extension` package. For that additional installation step is needed:

```
cd node/packages/aws-lambda-otel-extension
npm install
```